tests/Agent/UnitTests/Core.UnitTest/CrossAgentTests/CatMapTests.cs

```diff
@@ -124,7 +124,11 @@ public void Test(TestCase testCase)
             {
                 var transactionName = GetTransactionNameFromString(request.OutboundTxnName);
                 _transaction.CandidateTransactionName.TrySet(transactionName, (TransactionNamePriority)namePriority++);
+#if NETFRAMEWORK
                 var outboundHeaders = _agent.CurrentTransaction.GetRequestMetadata().ToDictionary();
+#else
+                var outboundHeaders = Enumerable.ToDictionary(_agent.CurrentTransaction.GetRequestMetadata());
```

Comment:
Why do we need something different here? Did the ToDictionary method go away?

Reply:
netstandard doesn't have `.ToDictionary()`, so we use the `MoreLinq` package to provide it. 

net8 does have `.ToDictionary()` but it can't be invoked as an extension method because of widespread use of other `MoreLinq` extensions throughout this test. 

My preference would be to dump `MoreLinq` entirely but I wasn't going to include that in this set of changes.